cardano-crypto-praos/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog for `cardano-crypto-praos`
 
+# Next version
+
+* Prefixed private bundled c functions with `cardano_` to ensure they are not
+  silently overwritten.
+
 ## 2.1.2.1
 
 *
```

Comment:
We don't have `Next version` in `cardano-ledger` and `cardano-base`. This allows us to cut a release from `master` at any time without any special preceding PRs

See this section about the [CHANGELOG](https://github.com/IntersectMBO/cardano-base/blob/master/RELEASING.md#changelogmd)

Also could you please bump up the version in the cabal file
```suggestion
## 2.2.0.0

* Prefixed private bundled c functions with `cardano_` to ensure they are not
  silently overwritten.
```